manuscript/draft.Rmd

```diff
@@ -305,8 +305,8 @@ $$y_t = \mu + \epsilon_t.$$
 
 Predictions from the "average" model are thus centered on $\mu$, which could
 either be the mean of the raw training richness values, or an output from the
-observer model.  This model's confidence intervals have a constant width that
-depends on the standard deviation of $\epsilon$, which can either be the
+observer model. The confidence iterval width in this model is constant and a 
+consequence of the standard deviation $\epsilon$, which can either be the
```

Comment:
"iterval" -> "interval", but I'm not sure I love either of these versions. What about:

> In this model the confidence intervals have a constant width that is set by the standard deviation...